kas/machine/rubikpi3.yml

```diff
@@ -17,6 +17,6 @@ repos:
 
 local_conf_header:
   machine-avocado-rubikpi3: |
-    AVOCADO_ALLOW_ROOT_LOGIN = "1"
+    #AVOCADO_ALLOW_ROOT_LOGIN = "1"
```

Comment:
Lets remove this key from this config completely and depend on this being set in base.yaml. I'd also like to update base.yaml to allow this option to be set from an env var, allowing maintainers to configure certain build boxes to always produce builds with early login.